package.json

```diff
@@ -72,7 +72,12 @@
     "nextjs:build": "pnpm --filter design-system-nextjs-integration build",
     "nextjs:lint": "pnpm --filter design-system-nextjs-integration lint",
     "changeset:publish": "pnpm changeset publish",
-    "changeset:version": "pnpm changeset version && pnpm install --lockfile-only"
+    "changeset:version": "pnpm changeset version && pnpm install --lockfile-only",
+    "primeng": "pnpm primeng:start",
```

Comment:
I'm getting the following errors when trying to run this command (I did a clean bootstrap before):

```
philipp@Philipps-MacBook-Pro design-system % pnpm primeng

> @swisspost/design-system@ primeng /Users/philipp/Projects/design-system
> pnpm primeng:start


> @swisspost/design-system@ primeng:start /Users/philipp/Projects/design-system
> pnpm --filter design-system-styles-primeng-workspace start


> @swisspost/design-system-styles-primeng-workspace@0.0.0 start /Users/philipp/Projects/design-system/packages/styles-primeng-workspace
> npm-run-all --parallel watch:lib serve:app


> @swisspost/design-system-styles-primeng-workspace@0.0.0 watch:lib /Users/philipp/Projects/design-system/packages/styles-primeng-workspace
> ng build --watch --configuration development styles-primeng


> @swisspost/design-system-styles-primeng-workspace@0.0.0 serve:app /Users/philipp/Projects/design-system/packages/styles-primeng-workspace
> ng serve --port 9325

Building Angular Package

------------------------------------------------------------------------------
Building entry point '@swisspost/design-system-styles-primeng'
------------------------------------------------------------------------------
✔ Compiling with Angular sources in Ivy full compilation mode.
✖ Generating FESM bundles
ERROR: Could not resolve entry module "dist/styles-primeng/esm2022/swisspost-design-system-styles-primeng.mjs".

Compilation failed. Watching for file changes...
✔ Browser application bundle generation complete.

Initial Chunk Files | Names         |  Raw Size
vendor.js           | vendor        |   5.70 MB | 
polyfills.js        | polyfills     | 336.73 kB | 
styles.js           | styles        | 246.20 kB | 
main.js             | main          |  42.62 kB | 
runtime.js          | runtime       |   6.54 kB | 

                    | Initial Total |   6.32 MB

Build at: 2024-03-20T10:36:35.167Z - Hash: b289c943ab41d1f4 - Time: 2268ms

./projects/styles-primeng-showcase/src/styles.scss - Error: Module build failed (from ../../node_modules/.pnpm/sass-loader@13.3.2_sass@1.64.1_webpack@5.88.2/node_modules/sass-loader/dist/cjs.js):
Can't find stylesheet to import.
  ╷
2 │ @use "primeng-theme";
  │ ^^^^^^^^^^^^^^^^^^^^
  ╵
  projects/styles-primeng-showcase/src/styles.scss 2:1  root stylesheet

./projects/styles-primeng-showcase/src/styles.scss?ngGlobalStyle - Error: Module build failed (from ../../node_modules/.pnpm/mini-css-extract-plugin@2.7.6_webpack@5.88.2/node_modules/mini-css-extract-plugin/dist/loader.js):
HookWebpackError: Module build failed (from ../../node_modules/.pnpm/sass-loader@13.3.2_sass@1.64.1_webpack@5.88.2/node_modules/sass-loader/dist/cjs.js):
Can't find stylesheet to import.
  ╷
2 │ @use "primeng-theme";
  │ ^^^^^^^^^^^^^^^^^^^^
  ╵
  projects/styles-primeng-showcase/src/styles.scss 2:1  root stylesheet



** Angular Live Development Server is listening on localhost:9325, open your browser on http://localhost:9325/ **


✖ Failed to compile.

```

Reply:
@gfellerph do you still have this error?

Reply:
nope, resolved